aws_topology/stackstate_checks/aws_topology/resources/elb_classic.py

```diff
@@ -13,78 +42,105 @@ class ELBClassicCollector(RegisteredResourceCollector):
     COMPONENT_TYPE = "aws.elb_classic"
     CLOUDFORMATION_TYPE = "AWS::ElasticLoadBalancing::LoadBalancer"
 
+    @set_required_access_v2("elasticloadbalancing:DescribeTags")
+    def collect_tag_page(self, elb_names):
+        if len(elb_names) <= 20:
+            return self.client.describe_tags(LoadBalancerNames=elb_names).get("TagDescriptions", [])
+        else:
+            return []
+
+    @set_required_access_v2("elasticloadbalancing:DescribeLoadBalancers")
+    def collect_elb_page(self, **kwargs):
+        # This doesn't use the utils.client_array_operation function as we need control over grabbing one page at a time
+        for elb_page in self.client.get_paginator("describe_load_balancers").paginate(
+            PaginationConfig={
+                "MaxItems": 20,  # Use 20 to match the describe_tags API call, where max items in one call is also 20
```

Comment:
I'm not sure whether we want to put a limitation on this as well. We can probably still fetch more here and batch them into 20 for the tag collection